lua/conform/health.lua

```diff
@@ -114,12 +114,25 @@ M.show_window = function()
       )
     else
       local filetypes = get_formatter_filetypes(formatter.name)
-      local line = string.format("%s ready (%s)", formatter.name, table.concat(filetypes, ", "))
+      local line = string.format(
+        "%s ready (%s) %s",
+        formatter.name,
+        table.concat(filetypes, ", "),
+        formatter.path
+      )
       table.insert(lines, line)
       table.insert(
         highlights,
         { "DiagnosticInfo", #lines, formatter.name:len(), formatter.name:len() + 6 }
       )
+      if formatter.path ~= nil then
+        table.insert(highlights, {
+          "DiagnosticInfo",
+          #lines,
+          formatter.name:len() + 7 + table.concat(filetypes, ", "):len() + 3,
```

Comment:
Let's pull the `table.concat(filetypes, ", ")` into a variable so we don't have to run it twice

Reply:
Will do